### ISSUE TITLE
1.13.1: build fails

### ISSUE THREAD

**Issue by kloczek** [openucx/ucx#8841]
### Describe the bug
Looks like with gcc 13.0.1 from Fedora rawhe nuild fails.

### Steps to Reproduce
- clone repo from 1.13.1 taf
- run
```
autoreconf -fiv
%configure \
        --disable-assertions \
        --disable-debug \
        --disable-logging \
        --disable-compiler-opt \
        --disable-optimizations \
        --disable-params-check \
        --disable-static \
        --enable-cma \
        --enable-gtest \
        --with-rdmacm \
        --with-verbs \
        --with-fuse3 \
        --with-ib-hw-tm \
        --without-cuda \
        --without-gdrcopy \
        --without-java \
        --without-knem \
        --without-rocm \
        --without-ugni \
        --withut-xpmem
```
- build

- OS version x86/64

Build fails with:
<details>

```console
[tkloczko@devel-g2v ucx-1.13.1]$ make -k
make  all-recursive
make[1]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1'
Making all in src/ucm
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucm'
Making all in .
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucm'
make[3]: Nothing to be done for 'all-am'.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucm'
Making all in cuda
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucm/cuda'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucm/cuda'
Making all in rocm
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucm/rocm'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucm/rocm'
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucm'
Making all in src/ucs
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucs'
Making all in vfs/sock
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucs/vfs/sock'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucs/vfs/sock'
Making all in .
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucs'
  CC       debug/libucs_la-debug.lo
In file included from debug/debug.c:27:
debug/debug.c: In function 'load_file':
debug/debug.c:275:53: error: 'PTR' undeclared (first use in this function)
  275 |     symcount = bfd_read_minisymbols(file->abfd, 0, (PTR)&file->syms, &size);
      |                                                     ^~~
debug/debug.c:275:53: note: each undeclared identifier is reported only once for each function it appears in
make[3]: *** [Makefile:1624: debug/libucs_la-debug.lo] Error 1
make[3]: Target 'all-am' not remade because of errors.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucs'
Making all in vfs/fuse
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucs/vfs/fuse'
make[3]: *** No rule to make target '../../../../src/ucs/libucs.la', needed by 'libucs_fuse.la'.
make[3]: Target 'all' not remade because of errors.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucs/vfs/fuse'
Making all in signal
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucs/signal'
make[3]: *** No rule to make target '../../../src/ucs/libucs.la', needed by 'libucs_signal.la'.
make[3]: Target 'all' not remade because of errors.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucs/signal'
make[2]: *** [Makefile:1946: all-recursive] Error 1
make[2]: Target 'all' not remade because of errors.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucs'
Making all in src/uct
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct'
Making all in .
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct'
  CC       base/libuct_la-uct_md.lo
base/uct_md.c:455:1: error: conflicting types for 'uct_md_mem_advise' due to enum/integer mismatch; have 'ucs_status_t(struct uct_md *, void *, void *, size_t,  unsigned int)' {aka 'ucs_status_t(struct uct_md *, void *, void *, long unsigned int,  unsigned int)'} [-Werror=enum-int-mismatch]
  455 | uct_md_mem_advise(uct_md_h md, uct_mem_h memh, void *addr, size_t length,
      | ^~~~~~~~~~~~~~~~~
In file included from base/uct_component.h:11,
                 from base/uct_md.h:14,
                 from base/uct_md.c:13:
/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/api/uct.h:2412:14: note: previous declaration of 'uct_md_mem_advise' with type 'ucs_status_t(struct uct_md *, void *, void *, size_t,  uct_mem_advice_t)' {aka 'ucs_status_t(struct uct_md *, void *, void *, long unsigned int,  uct_mem_advice_t)'}
 2412 | ucs_status_t uct_md_mem_advise(uct_md_h md, uct_mem_h memh, void *addr,
      |              ^~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
make[3]: *** [Makefile:926: base/libuct_la-uct_md.lo] Error 1
make[3]: *** No rule to make target '../../src/ucs/libucs.la', needed by 'libuct.la'.
make[3]: Target 'all-am' not remade because of errors.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct'
Making all in cuda
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/cuda'
make[4]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/cuda'
make[4]: Nothing to be done for 'all-am'.
make[4]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/cuda'
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/cuda'
Making all in ib
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/ib'
Making all in .
make[4]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/ib'
make[4]: *** No rule to make target '../../../src/ucs/libucs.la', needed by 'libuct_ib.la'.
make[4]: *** No rule to make target '../../../src/uct/libuct.la', needed by 'libuct_ib.la'.
make[4]: Target 'all-am' not remade because of errors.
make[4]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/ib'
Making all in rdmacm
make[4]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/ib/rdmacm'
make[4]: *** No rule to make target '../../../../src/ucs/libucs.la', needed by 'libuct_rdmacm.la'.
make[4]: *** No rule to make target '../../../../src/uct/libuct.la', needed by 'libuct_rdmacm.la'.
make[4]: *** No rule to make target '../../../../src/uct/ib/libuct_ib.la', needed by 'libuct_rdmacm.la'.
make[4]: Target 'all' not remade because of errors.
make[4]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/ib/rdmacm'
make[3]: *** [Makefile:1296: all-recursive] Error 1
make[3]: Target 'all' not remade because of errors.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/ib'
Making all in rocm
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/rocm'
make[4]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/rocm'
make[4]: Nothing to be done for 'all-am'.
make[4]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/rocm'
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/rocm'
Making all in sm
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm'
Making all in scopy
make[4]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm/scopy'
Making all in cma
make[5]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm/scopy/cma'
make[5]: *** No rule to make target '../../../../../src/ucs/libucs.la', needed by 'libuct_cma.la'.
make[5]: *** No rule to make target '../../../../../src/uct/libuct.la', needed by 'libuct_cma.la'.
make[5]: Target 'all' not remade because of errors.
make[5]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm/scopy/cma'
Making all in knem
make[5]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm/scopy/knem'
make[5]: Nothing to be done for 'all'.
make[5]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm/scopy/knem'
make[5]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm/scopy'
make[5]: Nothing to be done for 'all-am'.
make[5]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm/scopy'
make[4]: *** [Makefile:514: all-recursive] Error 1
make[4]: Target 'all' not remade because of errors.
make[4]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm/scopy'
Making all in mm
make[4]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm/mm'
Making all in xpmem
make[5]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm/mm/xpmem'
make[5]: Nothing to be done for 'all'.
make[5]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm/mm/xpmem'
make[5]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm/mm'
make[5]: Nothing to be done for 'all-am'.
make[5]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm/mm'
make[4]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm/mm'
make[4]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm'
make[4]: Nothing to be done for 'all-am'.
make[4]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm'
make[3]: *** [Makefile:514: all-recursive] Error 1
make[3]: Target 'all' not remade because of errors.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/sm'
Making all in ugni
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/ugni'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/ugni'
make[2]: *** [Makefile:1183: all-recursive] Error 1
make[2]: Target 'all' not remade because of errors.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct'
Making all in src/ucp
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucp'
make[2]: *** No rule to make target '../ucs/libucs.la', needed by 'libucp.la'.
make[2]: *** No rule to make target '../uct/libuct.la', needed by 'libucp.la'.
make[2]: Target 'all' not remade because of errors.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucp'
Making all in src/tools/vfs
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/vfs'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/vfs'
Making all in src/tools/info
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/info'
make  all-am
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/info'
make[3]: *** No rule to make target '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/libuct.la', needed by 'ucx_info'.
make[3]: *** No rule to make target '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucp/libucp.la', needed by 'ucx_info'.
make[3]: *** No rule to make target '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucs/libucs.la', needed by 'ucx_info'.
make[3]: Target 'all-am' not remade because of errors.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/info'
make[2]: *** [Makefile:506: all] Error 2
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/info'
Making all in src/tools/perf
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/perf'
Making all in cuda
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/perf/cuda'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/perf/cuda'
Making all in rocm
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/perf/rocm'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/perf/rocm'
Making all in lib
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/perf/lib'
make[3]: *** No rule to make target '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/libuct.la', needed by 'libucxperf.la'.
make[3]: *** No rule to make target '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucp/libucp.la', needed by 'libucxperf.la'.
make[3]: *** No rule to make target '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucs/libucs.la', needed by 'libucxperf.la'.
make[3]: Target 'all' not remade because of errors.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/perf/lib'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/perf'
make[3]: *** No rule to make target '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/uct/libuct.la', needed by 'ucx_perftest'.
make[3]: *** No rule to make target '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucp/libucp.la', needed by 'ucx_perftest'.
make[3]: *** No rule to make target '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/ucs/libucs.la', needed by 'ucx_perftest'.
make[3]: *** No rule to make target 'lib/libucxperf.la', needed by 'ucx_perftest'.
make[3]: Target 'all-am' not remade because of errors.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/perf'
make[2]: *** [Makefile:781: all-recursive] Error 1
make[2]: Target 'all' not remade because of errors.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/perf'
Making all in src/tools/profile
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/profile'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/src/tools/profile'
Making all in bindings/go
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/bindings/go'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/bindings/go'
Making all in bindings/java
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/bindings/java'
Making all in src/main/native
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/bindings/java/src/main/native'
make  all-am
make[4]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/bindings/java/src/main/native'
make[4]: Nothing to be done for 'all-am'.
make[4]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/bindings/java/src/main/native'
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/bindings/java/src/main/native'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/bindings/java'
make[3]: Nothing to be done for 'all-am'.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/bindings/java'
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/bindings/java'
Making all in test/apps
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/apps'
Making all in profiling
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/apps/profiling'
make[3]: *** No rule to make target '../../../src/ucs/libucs.la', needed by 'ucx_profiling'.
make[3]: Target 'all' not remade because of errors.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/apps/profiling'
Making all in iodemo
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/apps/iodemo'
make[3]: *** No rule to make target '../../../src/ucs/libucs.la', needed by 'io_demo'.
make[3]: *** No rule to make target '../../../src/uct/libuct.la', needed by 'io_demo'.
make[3]: *** No rule to make target '../../../src/ucp/libucp.la', needed by 'io_demo'.
make[3]: Target 'all' not remade because of errors.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/apps/iodemo'
Making all in sockaddr
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/apps/sockaddr'
  CXX      sa-sa_base.o
In file included from sa_base.h:10,
                 from sa_base.cc:7:
sa_util.h:62:9: error: ‘uint32_t’ does not name a type
   62 |         uint32_t ev_flags;
      |         ^~~~~~~~
sa_util.h:14:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
   13 | #include <vector>
  +++ |+#include <cstdint>
   14 |
sa_util.h:67:22: error: ‘uint32_t’ has not been declared
   67 |     void add(int fd, uint32_t ev_flags);
      |                      ^~~~~~~~
make[3]: *** [Makefile:576: sa-sa_base.o] Error 1
  CXX      sa-sa_main.o
In file included from sa_base.h:10,
                 from sa_main.cc:7:
sa_util.h:62:9: error: ‘uint32_t’ does not name a type
   62 |         uint32_t ev_flags;
      |         ^~~~~~~~
sa_util.h:14:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
   13 | #include <vector>
  +++ |+#include <cstdint>
   14 |
sa_util.h:67:22: error: ‘uint32_t’ has not been declared
   67 |     void add(int fd, uint32_t ev_flags);
      |                      ^~~~~~~~
make[3]: *** [Makefile:590: sa-sa_main.o] Error 1
  CXX      sa-sa_tcp.o
In file included from sa_base.h:10,
                 from sa_tcp.h:10,
                 from sa_tcp.cc:7:
sa_util.h:62:9: error: ‘uint32_t’ does not name a type
   62 |         uint32_t ev_flags;
      |         ^~~~~~~~
sa_util.h:14:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
   13 | #include <vector>
  +++ |+#include <cstdint>
   14 |
sa_util.h:67:22: error: ‘uint32_t’ has not been declared
   67 |     void add(int fd, uint32_t ev_flags);
      |                      ^~~~~~~~
sa_tcp.cc: In member function ‘virtual void tcp_worker::wait(const evpoll_set&, worker::conn_handler_t, worker::data_handler_t, int)’:
sa_tcp.cc:119:36: error: ‘struct evpoll_set::event’ has no member named ‘ev_flags’
  119 |             data_handler(ev.fd, ev.ev_flags);
      |                                    ^~~~~~~~
make[3]: *** [Makefile:604: sa-sa_tcp.o] Error 1
  CXX      sa-sa_util.o
In file included from sa_util.cc:7:
sa_util.h:62:9: error: ‘uint32_t’ does not name a type
   62 |         uint32_t ev_flags;
      |         ^~~~~~~~
sa_util.h:14:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
   13 | #include <vector>
  +++ |+#include <cstdint>
   14 |
sa_util.h:67:22: error: ‘uint32_t’ has not been declared
   67 |     void add(int fd, uint32_t ev_flags);
      |                      ^~~~~~~~
sa_util.cc:50:6: error: no declaration matches ‘void evpoll_set::add(int, uint32_t)’
   50 | void evpoll_set::add(int fd, uint32_t ev_flags) {
      |      ^~~~~~~~~~
sa_util.h:67:10: note: candidate is: ‘void evpoll_set::add(int, int)’
   67 |     void add(int fd, uint32_t ev_flags);
      |          ^~~
sa_util.h:58:7: note: ‘class evpoll_set’ defined here
   58 | class evpoll_set : public file_desc {
      |       ^~~~~~~~~~
sa_util.cc: In member function ‘void evpoll_set::wait(std::vector<event>&, int) const’:
sa_util.cc:73:66: error: too many initializers for ‘evpoll_set::event’
   73 |             event ev = { ev_array[i].data.fd, ev_array[i].events };
      |                                                                  ^
make[3]: *** [Makefile:618: sa-sa_util.o] Error 1
make[3]: Target 'all' not remade because of errors.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/apps/sockaddr'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/apps'
make[3]: *** No rule to make target '../../src/ucp/libucp.la', needed by 'test_ucp_config'.
make[3]: *** No rule to make target '../../src/ucs/libucs.la', needed by 'test_memtrack_limit'.
make[3]: Target 'all-am' not remade because of errors.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/apps'
make[2]: *** [Makefile:955: all-recursive] Error 1
make[2]: Target 'all' not remade because of errors.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/apps'
Making all in examples
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/examples'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/examples'
Making all in cmake
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/cmake'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/cmake'
Making all in test/gtest
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/gtest'
Making all in ucs/test_module
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/gtest/ucs/test_module'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/gtest/ucs/test_module'
Making all in ucm/test_dlopen
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/gtest/ucm/test_dlopen'
Making all in rpath-subdir
make[4]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/gtest/ucm/test_dlopen/rpath-subdir'
make[4]: Nothing to be done for 'all'.
make[4]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/gtest/ucm/test_dlopen/rpath-subdir'
make[4]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/gtest/ucm/test_dlopen'
make[4]: Nothing to be done for 'all-am'.
make[4]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/gtest/ucm/test_dlopen'
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/gtest/ucm/test_dlopen'
Making all in common/googletest
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/gtest/common/googletest'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/gtest/common/googletest'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/gtest'
make[3]: *** No rule to make target '../../src/ucs/libucs.la', needed by 'gtest'.
make[3]: *** No rule to make target '../../src/uct/libuct.la', needed by 'gtest'.
make[3]: *** No rule to make target '../../src/ucp/libucp.la', needed by 'gtest'.
make[3]: *** No rule to make target '../../src/tools/perf/lib/libucxperf.la', needed by 'gtest'.
make[3]: *** No rule to make target '../../src/uct/ib/libuct_ib.la', needed by 'gtest'.
make[3]: Target 'all-am' not remade because of errors.
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/gtest'
make[2]: *** [Makefile:3320: all-recursive] Error 1
make[2]: Target 'all' not remade because of errors.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1/test/gtest'
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1'
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1'
make[1]: *** [Makefile:762: all-recursive] Error 1
make[1]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/ucx-1.13.1'
make: *** [Makefile:628: all] Error 2
```
</details>

**Comment by prj-** [comment]
I believe this is a duplicate of https://github.com/openucx/ucx/issues/8347 and it should be closed @yosefe.

**Comment by devreal** [comment]
This is not a duplicate and is not fixed. The error below (reported above, among others) still happens with UCX 1.14.1 release when compiling against GCC 13.1:

```
../../../src/uct/base/uct_md.c:559:1: error: conflicting types for 'uct_md_mem_advise' due to enum/integer mismatch; have 'ucs_status_t(struct uct_md *, void *, void *, size_t,  unsigned int)' {aka 'ucs_status_t(struct uct_md *, void *, void *, long unsigned int,  unsigned int)'} [-Werror=enum-int-mismatch]
  559 | uct_md_mem_advise(uct_md_h md, uct_mem_h memh, void *addr, size_t length,
      | ^~~~~~~~~~~~~~~~~
```

**Comment by lahwaacz** [comment]
@devreal That error is fixed by https://github.com/openucx/ucx/pull/8804
